setup.py

```diff
@@ -23,7 +23,7 @@ def read_file(filename):
 
 setup(
     name='kinto-dist',
-    version='23.1.2',
+    version='23.2.0.dev0',
```

Comment:
Is the "dev0" part here intentional?

Reply:
I hadn't seen that. I pushed a fix to re-prepare